sdk/src/instructions/event.ts

```diff
@@ -3,69 +3,55 @@ import { CubikSDK } from "..";
 import {
   CreateEventAccounts,
   CreateEventHandlerArgs,
-  CreateEventJoinAccounts,
-  InviteEventJoinAccounts,
+  EventParticipantCreateAccounts,
+  EventParticipantInviteAccounts,
+  EventTeamCreateAccounts,
+  EventTeamCreateArgs,
   UpdateEventAccounts,
   UpdateEventArgs,
-  UpdateEventStatusAccounts,
-  UpdateEventStatusArgs,
 } from "../types";
 
 export const event = (sdk: CubikSDK) => {
   return {
     create: async (
-      args: CreateEventHandlerArgs,
-      accounts: CreateEventAccounts
+      accounts: CreateEventAccounts,
+      args: CreateEventHandlerArgs
     ) => {
-      const ix = await sdk.program.methods
-        .createEvent(args.matchingPool, args.event_admin_signer)
+      return await sdk.program.methods
+        .eventCreate(args)
         .accounts(accounts)
         .instruction();
-
-      return ix;
     },
 
-    createEventJoin: async (accounts: CreateEventJoinAccounts) => {
-      const ix = await sdk.program.methods
-        .createEventJoin()
+    update: async (accounts: UpdateEventAccounts, args: UpdateEventArgs) => {
+      return await sdk.program.methods
+        .eventUpdate(args)
         .accounts(accounts)
         .instruction();
-
-      return ix;
     },
 
-    update: async (args: UpdateEventArgs, accounts: UpdateEventAccounts) => {
-      const ix = await sdk.program.methods
-        .updateEvent(args.matchingPool)
+    teamCreate: async (
+      accounts: EventTeamCreateAccounts,
+      args: EventTeamCreateArgs
+    ) => {
+      return await sdk.program.methods
+        .eventTeamCreate(args)
         .accounts(accounts)
         .instruction();
-      return ix;
     },
 
-    updateStatus: async (
-      args: UpdateEventStatusArgs,
-      accounts: UpdateEventStatusAccounts
-    ) => {
-      const ix = await sdk.program.methods
-        .updateEventJoinStatus(args.status)
-        .accounts({
-          authority: accounts.authority,
-          eventAccount: accounts.eventAccount,
-          eventJoinAccount: accounts.eventJoinAccount,
-          projectAccount: accounts.projectAccount,
-          subAdminAccount: accounts.subAdminAccount,
-          rent: accounts.rent,
-          systemProgram: accounts.systemProgram,
-        })
+    participantCreate: async (accounts: EventParticipantCreateAccounts) => {
+      return await sdk.program.methods
+        .eventParticipantCreate()
+        .accounts(accounts)
         .instruction();
-      return ix;
     },
-    inviteEventJoin: async (accounts: InviteEventJoinAccounts) => {
-      const ix = await sdk.program.methods
-        .inviteEventJoin()
+
```

Comment:
change to participant.invite rather than participantInvite

so it looks like
cubiksdk.event.participant.invite() in the code